mdapi/MetricsDiscoveryHelper.cpp

```diff
@@ -348,22 +349,36 @@ void MDHelper::GetMetricsFromReport(
     const uint32_t informationCount = m_MetricSet->GetParams()->InformationCount;
 
     results.resize( metricsCount + informationCount );
+	if ( maxValues != NULL ) (*maxValues).resize(metricsCount);
 
     uint32_t    outReportCount = 0;
-    TCompletionCode res = m_MetricSet->CalculateMetrics(
+	TCompletionCode res = MetricsDiscovery::CC_ERROR_GENERAL;
+	if( maxValues != NULL )
+    res = ((MetricsDiscovery::IMetricSet_1_5*)m_MetricSet)->CalculateMetrics(
```

Comment:
Does this require metrics capabilities 1.5?  I believe we only check for metrics capabilities 1.1 presently, so we'll need to handle the case where metrics capabilities 1.5 are unavailable.

Reply:
It does, but it's in the driver since over a year now.
We can always add a notice in the control description that this option needs a newer driver or custom built md from github.